client/components/FormattingBar/buttons.ts

```diff
@@ -126,6 +129,16 @@ export const citation = {
 	controls: nodeControls(ControlsFootnoteCitation, 'citation'),
 };
 
+export const reference = {
+	key: 'reference',
+	title: 'Reference',
+	icon: 'chevron-right',
```

Comment:
Fine to ignore this for now but we should either find a real icon for this button, or merge its behavior with the link button somehow.

Reply:
Yeah - I plan on wrapping that into my next PR!